src/main/java/net/minecraftforge/fml/common/registry/GameRegistry.java

```diff
@@ -288,6 +298,46 @@ public static int getFuelValue(ItemStack itemStack)
         }
         return fuelValue;
     }
+    
+    /**
+     * Links a set of parts to an {@link IPartFactory} that can produce them.
+     */
+    public static void registerPartFactory(IPartFactory factory, ResourceLocation... names)
+    {
+        MultipartRegistry.registerPartFactory(factory, names);
+    }
+    
+    /**
+     * Links a set of parts to an {@link IAdvancedPartFactory} that can produce them.
+     */
+    public static void registerPartFactory(IAdvancedPartFactory factory, ResourceLocation... names)
```

Comment:
Exactly what asie said. IPartFactory is there in case you don't need to handle both, to prevent code duplication or people routing one method to the other, which is a bit of a dirty hack since you'd need to cast null.